components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -142,30 +129,91 @@ - (UIColor *)titleColorForState:(UIControlState)state {
 
 #pragma mark - UIView
 
-// TODO(#7645): Temporary layout until
-// https://github.com/material-components/material-components-ios/issues/7645 lands
 - (void)layoutSubviews {
   [super layoutSubviews];
 
-  if (self.items.count == 0) {
-    return;
+  CGFloat availableWidth = self.bounds.size.width;
```

Comment:
Nit: Please use `CGRectGetWidth(self.bounds)` instead of accessing the bounds directly. Especially for UIScrollViews, the bounds origin may not be (0, 0).

Reply:
Done.